testlib/helper.go

```diff
@@ -95,7 +95,7 @@ func (h *MainHelper) Main(m *testing.M) {
 func (h *MainHelper) setupStore() {
 	driverName := os.Getenv("MM_SQLSETTINGS_DRIVERNAME")
 	if driverName == "" {
-		driverName = model.DATABASE_DRIVER_MYSQL
+		driverName = model.DATABASE_DRIVER_POSTGRES
```

Comment:
Why the change in defaults here?

Reply:
The idea is take advantage of the speed that can be reached with postgresql using the `fsync=off` and `full_page_writes=off` configuration to run the integration tests faster. (I run all the test suite, with coverage and all, in 15minutes locally).

Reply:
Ah, I didn't catch that subtlety in the PR title/description -- I've suggested a title rename to help clarify that for anyone else who sees this PR. Makes sense!

Reply:
As an aside, I'd still love to see us run the full set of tests against both MySQL and Postgres in CI, for example (presumably in parallel).

Reply:
Yes I think we can do both.

Reply:
I'll do it as 2 different PRs later, one moving back to MySQL the CI tests, and other adding a new task in the pipeline to tests against postgresql.